geoutils/georaster.py

```diff
@@ -1661,4 +1664,34 @@ def split_bands(self, copy: bool = False, subset: Optional[Union[list[int], int]
                 raster.nbands = 1
                 bands.append(raster)
 
-        return bands if len(bands) > 1 else bands[0]
\ No newline at end of file
+        return bands if len(bands) > 1 else bands[0]
+    
+    def polygonize(self, in_value=1):
+        """
+        Return a GeoDataFrame polygonized from a raster.
+        :param in_value: Value of the raster from which to create geometries (Default is 1)
+        :type in_value: int, float
+        :returns: GeoDataFrame containing the polygonized geometries
```

Comment:
Should it return a GeoDataFrame or a Vector, @AdrienWehrle, @adehecq, @atedstone?

Reply:
I think a Vector would make more sense to be consistent with the geoutils' frame.